### ISSUE TITLE
[feature request] Research page card new design

### ISSUE THREAD

**Issue by BaltacMihai** [ONEARMY/community-platform#2450]
**Is your feature request related to a problem? Please describe.**

You may find it difficult to understand the cards on the Research page since there are many elements as you scroll, namely the right portion with the three numbers (Useful, comments, and updates).

**Describe the solution you'd like**

In order to check whether there was something similar or how the issue was solved, I also looked at the other pages.
One solution would be to relocate the top icons from the "table header" to each card and add a tooltip that would explain each icon's meaning when the mouse is hovered over it. This is how we address the issue of visibility.

**Describe alternatives you've considered**

Why pagination isn't utilized for "tables" so we may restrict the number of entries per page is one thing I don't understand.

Therefore, pagination is used to reduce the amount of entries

**Additional context**
I quickly mocked up the version I was envisioning.
![Group 98](https://github.com/ONEARMY/community-platform/assets/72079422/f6c9333b-0a9c-44e7-bdac-a2efb18f9a1f)

Additionally, I'm not sure why the user needs to know how often it was updated when he only needs to know when the most recent update occurred.

**Comment by davehakkens** [comment]
Makes sense to me. Also matches the layout with mobile and the how-to page.
Do need to figure out the icon for `Updates` (this is used so you can see if a research is big and in depth with many stepts or not)

Are you able to code it @BaltacMihai?

**Comment by BaltacMihai** [comment]
I got it I had a look at the design system and saw some icons. I'm not sure if we utilize an icon collection that is free for commercial use or if we created our own.I can design one or two concepts of icon. Also, for the icons I have a suggestion to improve the overall layout, where should I put the suggestions? In a new issue?

I can work on it to get it implemented, but first I want to make sure I have the appropriate icon.
